### ISSUE TITLE
Variable sized mini-batches

### ISSUE THREAD

**Issue by pskrunner14** [pskrunner14/translator#1]
Model can't handle variable size mini-batches

**Comment by pskrunner14** [comment]
Add `torch.nn.utils.rnn.pad_sequence` as preprocessing part and use packing and padding in `encoder`
